libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -855,11 +855,33 @@ void AC_PosControl::input_vel_accel_z(float &vel, const float accel, bool ignore
 
 /// set_pos_target_z_from_climb_rate_cm - adjusts target up or down using a commanded climb rate in cm/s
 ///     using the default position control kinematic path.
+///     The zero target altitude is varied to follow pos_offset_z
+void AC_PosControl::set_pos_target_z_from_climb_rate_cm(const float vel, float pos_offset_z)
```

Comment:
Let's skip the "const" in front of float aguments.  It just makes it more difficult to read.